FhirToDataLake/azurepipelines/pack-nugets.yml

```diff
@@ -0,0 +1,30 @@
+steps:
+- checkout: self
+
+- download: current
+  artifact: 'native-linux-x64'
+
+- download: current
+  artifact: 'native-windows-x64'
+
+- script: |
+    mkdir -p src/Microsoft.Health.Fhir.Synapse.Parquet/native/linux-x64
+    mkdir -p src/Microsoft.Health.Fhir.Synapse.Parquet/native/windows-x64
+    cp -r $(Pipeline.Workspace)/native-linux-x64/** src/Microsoft.Health.Fhir.Synapse.Parquet/native/linux-x64
+    cp -r $(Pipeline.Workspace)/native-windows-x64/** src/Microsoft.Health.Fhir.Synapse.Parquet/native/windows-x64
+    dotnet pack src/Microsoft.Health.Fhir.Synapse.Parquet/Microsoft.Health.Fhir.Synapse.Parquet.csproj -p:NuspecFile=BuildParquet.nuspec -p:NuspecProperties="version=$(version)" -c Release -o '$(Build.ArtifactStagingDirectory)/nupkgs'
+  workingDirectory: FhirToDataLake
+  displayName: Nuget pack
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    pathToPublish: '$(Build.ArtifactStagingDirectory)/nupkgs'
```

Comment:
Just a discussion, maybe we can move Parquet project in more separated location? Now the other codes refer Parquet Nuget package, feel like it should be placed in separated location, and be managed alone.

Reply:
Good advice, removed all native lib related code to a seperate folder